sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -622,7 +624,7 @@ case class DescribeTableCommand(
     partitionSpec: TablePartitionSpec,
     isExtended: Boolean,
     override val output: Seq[Attribute])
-  extends DescribeCommandBase {
+  extends DescribeCommandBase with CatalogSerializer {
```

Comment:
If we keep `toLinkedHashMap` method in `CatalogTable` and its friends, we don't need to change this command at all.